api/HTMLInputElement.json

```
@@ -2201,6 +2201,57 @@
           }
         }
       },
+      "showPicker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/showPicker",
+          "spec_url": "https://html.spec.whatwg.org/multipage/input.html#dom-input-showpicker",
+          "support": {
+            "chrome": {
+              "version_added": "99"
+            },
+            "chrome_android": {
+              "version_added": "99"
+            },
+            "edge": {
+              "version_added": false
```

I assume this will work in Edge as well?

```suggestion
              "version_added": "99"
```